src/front-end/typescript/lib/pages/opportunity/sprint-with-us/edit/index.tsx

```diff
@@ -97,7 +97,7 @@ function makeInit<K extends Tab.TabId>(): component_.page.Init<
               routeParams.opportunityId,
               (response) => response
             ),
-            tabId === "proposals"
+            (["proposals", "teamQuestions", "codeChallenge", "teamScenario"] as Tab.TabId[]).includes(tabId)
```

Comment:
Not sure if there's a more elegant way of doing this check?

Reply:
Code looks good to me. It might be worthwhile abstracting this check to a function like `shouldLoadProposalsForTab(tabId: TabId): boolean` either in this module or in `.../edit/tab/index.ts`.